### PR TITLE
Rewrite ig-list

### PR DESCRIPTION
**Pull request by petko5kov**

### CODE REVIEW

**Comment by petko5kov** [comment]
Few test that failed are commented just to update to Angular 2 RC4. They will be handled in the test task.

File: src/list/list.ts [hunk 1]
```diff
@@ -97,26 +101,25 @@ export class Item implements OnDestroy {
...
 
-    private panMove = (ev: Object) => {
+    private panMove = (ev: HammerInput) => {
         /*if (!ev.additionalEvent) {
             return;
         }*/
 
         if (this.cancelEvent(ev)) return;
 
+        console.log(this.getLeftPosition());
```
Comment (kdinev) [inline src/list/list.ts]: Maybe get rid of this?

**Comment by kdinev** [comment]
@petko5kov Merge master to resolve the conflicts and I will merge the pull request.

**Comment by petko5kov** [comment]
Rewrite ig-list: improve its pan functionality; implement it in angular way

**Pull request merged**